contents/handbook/growth/sales/sales-operations.md

```diff
@@ -6,120 +6,66 @@ showTitle: true
+### What about Scale Free?
 
-People currently come into HubSpot through one of 3 ways:
-- They email hey@posthog.com, sales@posthog.com or another email address if we have created a custom one for a specific group
-- They sign up to the PostHog app
-- They are manually added to HubSpot by a member of the team, e.g. if you met someone interested in PostHog at an event
+We have recently started rolling out Scale Free. Our plan is to make it widely available as a config option in Open Source, but we're keeping them separate for now. Scale Free will be appealing to customers who need Clickhouse due to volume rather than Postgres, but are happy with 3 logins only and community-based support. 
 
-### Email
+We have a short waiting list of customers waiting for deployment, so are currently prioritising those with 10k-1m MAUs, are on GCP and are familiar with Helm Charts/k8s. 
 
-New conversations come into 'Unassigned', whereas ongoing conversations will go straight to your inbox.
+## FAQs
 
-We do not have super defined roles here, but generally:
-- James H deals with Enterprise queries
-- Yakko takes care of Startup queries
-- Paolo focuses on existing customers
-- Charles oversees sales ops and HubSpot admin
+_Can I give a Scale customer a free trial?_
 
-However, anyone can and should jump in if they can help or they see someone hasn't been responded to, especially when folks are on holiday!
+No, because we don't need to - they can get up and running with our Scale Free or Cloud plans first if they want to try out PostHog for free. You'll find a lot of inbound customers will do this anyway before talking to us about Scale. 
 
-We have lots of handy templates you can use as well - just select _Templates_ in the email window in Hubspot. If you find yourself sending the same type of email repeatedly, you may want to create your own template - go to 'Conversations' -> 'Templates'.
+_Can I give a Scale customer a discount?_
 
-If an inbound email is about one of our Startup or Enterprise plans, you should create a Deal - more on this below. 
+Again, no need - we already have usage-based pricing which is _heavily_ discounted at higher volumes, and we only bill month-to-month, so customers don't need to feel locked in to a longer term contract. 
 
-In addition to hey@posthog.com and sales@posthog.com, we sometimes create special one-off email addresses to use for specific groups, such as for an event or promotion. If you create a Google group and you want messages to flow into HubSpot to be managed, make sure you add our [HubSpot inbox email address](hello-1@posthoginc.hs-inbox.com) to your group as a member.
+_How do I work with a customer who wants to sign an MSA?_
 
-### New PostHog signups
+This occasionally happens when we are dealing with very large companies, who may prefer to sign an MSA due to their internal procurement processes. We have a contract version of our standard terms and conditions that we can use for this - ask Charles. 
```

Comment:
oops - covered above... perhaps just add a tiny bit more context (my response is what you'd say)